### DISCUSSION TITLE
테스트 API 서버 사용법

### DISCUSSION THREAD

**Discussion by kimminkyeu** [Kernel360/F2-BAGUNI#103] [메모장]
- API 명세 확인 (인증 필요 없음) - [Swagger](https://minlife.me/swagger-ui/index.html)
- 테스트 서버 주소 - [https://minlife.me/](https://minlife.me/)

### 사용법
1. 브라우저를 킨다.
2. [테스트 서버](https://minlife.me/)에 로그인해서 쿠키를 획득한다.
3. 브라우저에서 `access_token` 쿠키 값을 복사한다.
4. 프론트 로컬 환경에서 복사한 토큰을 `fetch`시 함께 전송한다.
    :warning: @Gyaak - 프론트 로컬앱 주소인 `http://localhost:3000` 과 `https://{아직 미정}.vercel.app` 을 CORS 에 추가해야 합니다.
   --> localhost:3000은 작업 완료했습니다.

**Comment by Gyaak** [comment]
> ⚠️ 프론트 로컬앱 주소인 http://localhost:3000 과 https://{아직 미정}.vercel.app 을 CORS 에 추가해야 합니다.
 --> localhost:3000은 작업 완료했습니다.

이슈등록 해주세요
프론트 배포 되면 수정할게요

**Reply by kimminkyeu** [reply]
확인 완료
